movie_rec/recommender_server.py

```diff
@@ -0,0 +1,66 @@
+import json
+from optparse import OptionParser
+import psycopg2
+
+class MovieRecommender(object):
```

Comment:
Should this class name match the filename? What's the difference between this and the existing `MovieRecommender`?